pkg/orchestrator/assetscanestimationwatcher/watcher.go

```diff
@@ -237,9 +284,20 @@ func (w *Watcher) reconcilePending(ctx context.Context, assetScanEstimation *mod
 
 	assetScanEstimation.State.LastTransitionTime = utils.PointerTo(time.Now())
```

Comment:
multiple call to `time.Now()` in the same flow, probably can be called only once.

Reply:
Now this times are not related :) they are being called in different phases in the code.